Lambdas/GHHooks/Models/Aliases.swift

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Always with the random blank lines even at the top of a file 🙂

Reply:
I haven't found a general rule for when to put a blank line and when not, but generally i don't like starting definitions right at the first line.